### DISCUSSION TITLE
AtomS3 Lite ESP32S3 Suitable?

### DISCUSSION THREAD

**Discussion by nolantern** [raomin/ESPAltherma#318] [Q&A] [answered]
Hi,

is a [AtomS3 Lite ESP32S3 ](https://shop.m5stack.com/products/atoms3-lite-esp32s3-dev-kit) suitable for ESPAltherma?

I ask because I want to try to make use of the [X85A smart grid / 4 level power limitation ](https://github.com/raomin/ESPAltherma/discussions/170) for my EHSX04P50EF.
The M5Stick lacks enough GPIO's.
The AtomS3 Lithe has 6 GPIO's and a [4 Relay Unit](https://shop.m5stack.com/products/4-relay-unit) can be connected.

**Comment by KalaNagHTD** [comment, marked as answer]
That might help: https://github.com/raomin/ESPAltherma/discussions/237
It‘s the Atom3, not the AtomS3
